### ISSUE TITLE
Allow wrapping cell text

### ISSUE THREAD

**Issue by TonyGermaneri** [TonyGermaneri/canvas-datagrid#134]
Also provide the properties:
Line height: (+/- font height)
Justification: left, right, center
review padding directives.

**Comment by ronbegley** [comment]
Mind if I ask how this is intended to work?

I'm doing this and it still acts like 'nowrap' is set

```
                  grid = canvasDatagrid({
                        parentNode: parentNode,
                        data: gridData,
                        style: {
                            cellWidth: 200,
                            cellWhiteSpace: 'normal'
                        }
                    });
```

**Comment by TonyGermaneri** [comment]
Looks like you have it set correctly.  Data is clipped at the size of the cell, if the cell is tall enough for more than 1 row, it will draw multiple rows.  Rows do not expand height to fit their content.

I think maybe you're expecting the rows to expand in height.  I can add that feature, but right now it's not automatic, you'll need to set the height of the row manually.  I used this as the default behavior for now because it is the behavior I observed in other grid programs.  Another behavior I'd like to add is to allow overflow to overflow the cell without increasing row height if there is no data in the lower cells.
